File: 백준/Bronze/31822. 재수강/재수강.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        String A = scan.next();
        String [] arr = new String[scan.nextInt()];
        
        for(int i=0; i<arr.length; i++) arr[i] = scan.next();
        
        int count = 0;
        for(int i=0; i<arr.length; i++) {
        	if(A.substring(0, 5).equals(arr[i].substring(0, 5))) {
        		count++;
        	}
        }
        System.out.println(count);
	}
}

File: 백준/Bronze/31822. 재수강/재수강.py
A = input()
arr_length = int(input())
    
arr = []
for _ in range(arr_length):
    arr.append(input())
        
count = 0
for i in range(arr_length):
    if A[:5] == arr[i][:5]:
        count += 1
            
print(count)